src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+
+webrisk.api.key=AIzaSyCGON0RcyFb-k2TfRfEBwTw5CS3g3sla4I
```

Comment:
Cześć - review zrobimy niedługo ale już teraz chciałem Ci dać znać, że ten klucz jest dość newralgiczny i ktoś może go zescrapować (repo jest publiczne) i nabić Ci (o ile nie ustawiłeś limitu) spory rachunek na koncie googla. Jeśli możesz to anuluj (wyłącz) ten klucz by nie był aktywny :slightly_smiling_face:

Reply:
Ustawiłem limit - 10 USD, 
Ale skoro mogę to go wyłączam :)